syntax = "proto3";

message Empty {
}

message UserID {
    string id = 1;
}